package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "2.0.0",
+  "version": "1.2.0",
```

Comment:
Probably can just manually change this back to 2.0.0 instead of reverting anything if we plan on squashing and merging.